Currently trouble shooting "prepare_inception.py"
